### PR TITLE
atomic: continue to run on failure

### PR DESCRIPTION
**Pull request by mike-nguyen**
This is a follow-on to #391 for the atomic test suite.  The changes
allow the tests to continue to run on failure using Ansible's
block/rescue/always.

### CODE REVIEW

File: tests/atomic/pull_digest.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+# set ft=ansible
+#
+
+- name: Set image name
+  set_fact:
+    fq_name: "docker.io/busybox"
+    digest:
```
Comment (miabbott) [inline tests/atomic/pull_digest.yml]: These two facts should probably be prefixed to make them unique to this playbook.

File: tests/atomic/pull_storage.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+# set ft=ansible
+#
+
+- name: Set image name
+  set_fact:
+    fq_name: "docker.io/busybox"
```
Comment (miabbott) [inline tests/atomic/pull_storage.yml]: This should be prefixed to make it unique to the playbook

File: tests/atomic/images_update.yml [hunk 3]
```diff
@@ -0,0 +1,132 @@
+---
+# set ft=ansible
+#
+
+- name: Set image name
+  set_fact:
+    fq_name: "docker.io/alpine"
+    local_name: "{{ ansible_docker0.ipv4.address}}:5000/alpine"
```
Comment (miabbott) [inline tests/atomic/images_update.yml]: These should be prefixed to make them unique to the playbook

File: tests/atomic/pull_tags.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
+---
+# set ft=ansible
+#
+
+- name: Set container name
+  set_fact:
+    fq_name: "docker.io/busybox"
+    short_name: "busybox"
+    tag: 1
```
Comment (miabbott) [inline tests/atomic/pull_tags.yml]: These should be prefixed to make them unique to the playbook

**Comment by miabbott** [comment]
I made a couple of comments about prefixing some variables to be unique, but I don't think it is absolutely necessary.

Since the tests are green and everything else LGTM, we could merge this without those changes.

Let me know if you want to change those variables

**Comment by mike-nguyen** [comment]
I'll make these changes and push a fixup.

**Comment by mike-nguyen** [comment]
Pushed fixup with prefixed variables

**Pull request merged**